tests/sequential_curriculum_test.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+from nle.env.tasks import NetHackScore, NetHackScout, NetHackStaircase
+
+from syllabus.core import make_multiprocessing_curriculum, make_ray_curriculum
+from syllabus.curricula import (AnnealingBoxCurriculum,
+                                CentralizedPrioritizedLevelReplay,
+                                DomainRandomization,
+                                LearningProgressCurriculum, NoopCurriculum,
+                                PrioritizedLevelReplay, SequentialCurriculum, Condition,
+                                SimpleBoxCurriculum)
+from syllabus.tests import (create_cartpole_env, create_nethack_env,
+                            get_test_values, run_native_multiprocess,
+                            run_ray_multiprocess, run_single_process, run_episode)
+
+N_ENVS = 1
+N_EPISODES = 34
+
+nethack_env = create_nethack_env()
+cartpole_env = create_cartpole_env()
+
+def create_sequential_curriculum_1(task_space):
+
+    curricula = []
+    stopping = []
+
+    # Stage 1 - Survival
+    stage1 = [0, 1, 2, 3]
+    stopping.append(
+        Condition(
+            metric_name="episode_return", comparator='>=', value=0.9
+        ) & Condition(
+            metric_name="episodes", comparator='>=', value=5000
+        )
+    )
+
+    # Stage 2 - Harvest Equipment
+    stage2 = [4, 5]
+    stopping.append(
+        Condition(
+            metric_name="episode_return", comparator='>=', value=0.9
+        ) & Condition(
+            metric_name="episodes", comparator='>=', value=5000
+        )
+    )
+
+    # Stage 3 - Equip Weapons
+    stage3 = [6, 7]
+
+    curricula = [stage1, stage2, stage3]
+    curriculum = SequentialCurriculum(curricula, stopping, task_space)
+
+    return curriculum
+
+def foo(obs, info):
+    return
+
+def create_sequential_curriculum_2(task_space):
+    curricula = []
+    stopping = []
+
+    # Stage 1 - Survival
+    stage1 = [0, 1, 2, 3]
+    stopping.append(
+        Condition(
```

Comment:
I like that this is structured, but I think it was easier to use as a string. Could you write some syntactic sugar so that we can pass in a string here, and then convert it to a Condition object within the curriculum?